packages/internals/src/__tests__/engine-commands/getConfig.test.ts

```diff
@@ -33,7 +35,7 @@ describe('getConfig', () => {
     expect(config.datasources[0].provider).toEqual('sqlite')
     expect(config.generators).toHaveLength(0)
     expect(config.warnings).toHaveLength(0)
-    expect(config).toMatchSnapshot()
+    expect(serialize(JSON.stringify(config, null, 2))).toMatchSnapshot()
```

Comment:
This is so that I could use the jest snapshot serializer, which was disabled in the jest config. I could try to enable it for all tests, but maybe there was a reason it was disabled?

Reply:
Thanks for clarifying. I'd argue that this is probably not the right PR to enable the jest serializer in all tests. Maybe a legacy thing to check at a later stage.